api/src/main/java/org/openmrs/module/pihcore/task/ClosePregnancyProgramTask.java

```diff
@@ -0,0 +1,90 @@
+package org.openmrs.module.pihcore.task;
+
+import org.joda.time.DateTime;
+import org.openmrs.Concept;
+import org.openmrs.PatientProgram;
+import org.openmrs.PatientState;
+import org.openmrs.Program;
+import org.openmrs.ProgramWorkflow;
+import org.openmrs.ProgramWorkflowState;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.pihcore.config.Config;
+import org.openmrs.module.pihcore.config.ConfigDescriptor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Patients should be unenrolled automatically from the pregnancy program if they have been in the Antenatal Treatment State for 11 months. The program outcome should be Lost to Follow Up.
+ * Patients should be unenrolled automatically from the pregnancy program 6 weeks after the Delivered treatment state start date with an outcome of Treatment Completed
+ */
+public class ClosePregnancyProgramTask implements Runnable {
+
+    private final Logger log = LoggerFactory.getLogger(getClass());
+    private static boolean isExecuting = false;
+
+    private static final String PREGNANCY_PROGRAM_UUID = "6a5713c2-3fd5-46e7-8f25-36a0f7871e12";
+
+    private static final String PREGNANCY_PROGRAM_TREATMENT_STATUS_WORKFLOW_UUID = "9a3f8252-1588-4f7b-b02c-9e99c437d4ef";
+
+    private static final String ANTENATAL_PROGRAM_STATE_UUID = "a83896bf-9094-4a3c-b843-e75509a52b32";
+
+    private static final String POSTPARTUM_PROGRAM_STATE_UUID = "a735b5f6-0b63-4d9a-ae2e-70d08c947aed";
```

Comment:
@mogoodrich , can you please use the PihEmrConfigConstants, instead of hard coding those uuids again here?